### DISCUSSION TITLE
Welcome to dosemu2 Discussions!

### DISCUSSION THREAD

**Discussion by stsp** [dosemu2/dosemu2#1362] [General]
Hi everyone!
Seems like we have a new discussion forums
where @ mentions supposed to work not only
with the team members.

Lets test:
@bolle732 do you see this message?

**Comment by andrewbird** [comment]
I wonder if you can convert issues such as #1335 to a discussion?

**Comment by stsp** [comment]
Indeed:
https://github.com/dosemu2/dosemu2/discussions/1335
Cool. :)

**Reply by andrewbird** [reply]
and good that the issue number didn't change!
